src/reducers/reducer-cities.js

```diff
@@ -0,0 +1,45 @@
+import { ADD_CITY } from '../actions';
+
+const DEFAULT_STATE = {
+  cityList: [],
+};
+
+const parseCity = (data) => {
+  const newCity = {
+    name: data.city.name,
+    temperature: [],
+    avgTemp: '',
+    pressure: [],
+    avgPressure: '',
+    humidity: [],
+    avgHumidity: '',
+  };
+  const average = (array) => array.reduce((a, b) => a + b) / array.length;
+
+  for (let i = 0; i < data.list.length; i += 1) {
```

Comment:
Would try to avoid for loops for the sake of readability - prefer `map` here if possible